dropwizard-cassandra/src/main/java/smartthings/dw/cassandra/CassandraHealthCheck.java

```diff
@@ -10,14 +11,13 @@
 public class CassandraHealthCheck extends NamedHealthCheck {
 	private final static Logger LOG = LoggerFactory.getLogger(CassandraHealthCheck.class);
 	private final Session session;
-	private final String validationQuery;
-	private final boolean validationQueryIdempotence;
+	private final SimpleStatement simpleStatement;
 
 	@Inject
 	public CassandraHealthCheck(Session session, CassandraConfiguration config) {
 		this.session = session;
-		this.validationQuery = config.getValidationQuery();
-		this.validationQueryIdempotence = config.getValidationQueryIdempotence();
+		this.simpleStatement = new SimpleStatement(config.getValidationQuery());
```

Comment:
Can we push `SimpleStatement` into the Config class itself?

If folks wanted to override it, they'd just wrap it in their own config that exposes all the specific switches they want to override.  Otherwise, we're in the same situation in that this OSS lib would need to support all the permutations of switches exposed.

Reply:
And not allow setting the query in the configuration as a string?

Reply:
Right.  At least that's what I'd propose to get us out of being the middle man.

They would just subclass Config to provide various things like query statement override / etc.  Which then could be used to build this Config.